src/ChorizoServer.cs

```diff
@@ -33,7 +33,7 @@ public class Chorizo
             SocketMachine = socketMachine ?? new DotNetSocketMachine();
             ProtocolConnectionHandler = protocolConnectionHandler ?? new HttpConnectionHandler
             {
-                SocketReader = new SocketReader(),
+                SocketReader = new InternalSocketReader(),
```

Comment:
What is an `InternalSocketReader`? It sounds like this is being used to read and write sockets on localhost only. If that's the case, ignore me, otherwise I wonder if the name could be clearer?